src/main/resources/static/js/index.js

```diff
@@ -1,139 +1,143 @@
+/*global currentDelimiter, currentHeaderLines, preview, optimzeforsource, resetConfig*/
 /**
- * Calls preview function after delimiter has been changed. Checks delimiter to be a valid regular expression.
- * @param del new value of the delimiter
+ * Calls preview function after delimiter has been changed. Checks delimiter to
+ * be a valid regular expression.
+ * 
+ * @param del
+ *            new value of the delimiter
  */
 function changedDelimiter(del) {
-    try {
-        new RegExp(del);
-        currentDelimiter = (del === null || del === "") ? "\n" : del;
-    } catch(e) {
-        currentDelimiter = "\n";
-    }
-    preview();
+	try {
+		var regex = new RegExp(del);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Found non-literal argument to RegExp Constructor](https://app.codacy.com/gh/FraunhoferIOSB/ChillImport/pullRequest?prid=4781409)